java-components/cache/src/main/resources/application.properties

```diff
@@ -8,53 +8,23 @@ quarkus.container-image.group=redhat-appstudio
 quarkus.container-image.name=hacbs-jvm-cache
 quarkus.jib.working-directory=/deployments
 quarkus.micrometer.enabled=true
-%dev.quarkus.kubernetes-client.trust-certs=true
-%dev.quarkus.log.category."com.redhat.hacbs.artifactcache".level=DEBUG
 quarkus.aws.devservices.localstack.image-name=localstack/localstack:0.11.5
 quarkus.http.limits.max-body-size=1000M
 quarkus.jib.user=0
+
+# default repo config
 build-policies=default
```

Comment:
The configuration.adoc says `By default this cache is configured to have a `default` and `central-only` configuration as follows: build-policies=default,central-only`  ; this seems inconsistent?